toybox.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: toybox
+  version: "0.8.12"
+  epoch: 0
+  description: "A BSD-licensed alternative to busybox"
+  copyright:
+    - license: BSD-0-Clause
+
+environment:
+  contents:
+    packages:
+      - bash
+      - binutils
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - wolfi-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/landley/toybox/archive/refs/tags/${{package.version}}.tar.gz
+      expected-sha512: b6984d475b253ecf2b2fdcb3107fab690f794442dd7e3cd8b9b598381e19eaaa0ab2c2d96f4f97b27b668c5c45e61d3bab5647015fa66314d8e565337f54d775
```

Comment:
we can use git-checkout pipeline in here?

Reply:
Working on this, thanks for the advice